deploy/ocs-operator/manifests/ocs-operator.clusterserviceversion.yaml

```diff
@@ -3281,8 +3281,6 @@ spec:
     name: ceph-container
   - image: quay.io/csiaddons/k8s-sidecar:v0.5.0
     name: csiaddons-sidecar
-  - image: quay.io/ocs-dev/ocs-must-gather:latest
-    name: ocs-must-gather
```

Comment:
We should not remove this. `RelatedImages` is a list of images that we use, even if we do not maintain it in this repo.

Reply:
+1